client/src/index.jsx

```diff
@@ -35,19 +53,24 @@ class App extends React.Component {
   }
   render() {
     return (
-      <MainContainer>
-        <Title>CONTENT FOR THIS GAME
-          <Browse className='note'>
-            <a href="https://google.com">
-              Browse all
-              <em>
-                ({this.state.data.length})
-              </em>
-            </a>
-          </Browse>
-        </Title>
-        <RelatedContent data={this.state.data} />
-      </MainContainer>
+      Object.keys(this.state.data[0]).length ?
+        <MainContainer>
+          <Title>CONTENT FOR THIS GAME
+            <Browse className='note'>
+              <StyledA href='https://google.com'>
+                {`Browse all `}
+                <em style={{ fontStyle: 'normal' }}>
+                  ({this.state.data.length})
+                </em>
+              </StyledA>
+            </Browse>
+          </Title>
+          <DlcList data={this.state.data} setHoverPosition={this.setHoverPosition} />
+          <Hover style={{ left: this.state.hover_left, top: this.state.hover_top }}>
```

Comment:
Is there a reason why some styling is done as styled components and in this file the styling is done this way?

Reply:
My hover box needs to be positioned relative to the view window, but I should probably use props for the styled components for consistency. Will refactor this!